### ISSUE TITLE
Improve color contrast of search bar on homepage of onrr.gov

### ISSUE THREAD

**Issue by AlexandracmONRR** [DOI-ONRR/onrr.gov-site#2463]
The search bar on the homepage does not pass 508 compliance. The text of "search" in the search bar does not contrast enough with the background color of the search rectangle.

**Comment by Maroyafaied** [comment]
@AlexandracmONRR please specify the color for @greensomes

**Comment by eelzi-ONRR** [comment]
@AlexandracmONRR Do you recall how you tested this and can you test again? I had trouble pulling the hex colors from the code but used an eyedropper tool and got that it passes<img width="896" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBekdWQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--3bb716b74b4a154be5aa1f90105e57966f6b0406/image.png" alt="image.png" />

**Comment by greensomes** [comment]
@eelzi-ONRR , @AlexandracmONRR , can this issue be closed? If not, please provide the color value for the text or background. Thanks!

**Comment by AlexandracmONRR** [comment]
@eelzi-ONRR - similarly I had pulled the hex color, since I wasn't sure of the color used in the search bar. From the color that I pulled, it failed.

<img width="791" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNG0vQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--254f7e6a458e7125ca33877be71d3a5564802051/image.png" alt="image.png" />

I think we should just make the color the same as the magnify glass, which i think is much closer to white.

**Comment by AlexandracmONRR** [comment]
@greensomes, can you change the text color of the search bar (before it is clicked) to: #f3f3f8

**Comment by AlexandracmONRR** [comment]
@greensomes - it looks great! Thank you!
